### DISCUSSION TITLE
Pinned tabs: only show favicon & horizontally

### DISCUSSION THREAD

**Discussion by DutchPete** [Floorp-Projects/Floorp#1261] [Feature Request]
The way pinned tabs are arranged is not very practical as they take up vertical space unnecessarily.

My request is to only show their favicon and arrange those horizontally to allow more vertical space for unpinned tabs.

![image](https://github.com/Floorp-Projects/Floorp/assets/10794265/7f13fbc2-69ec-436a-8253-4c3bcdad7175)

**Comment by Arindrew** [comment]
Came here to add this exact feature request. Otherwise a perfect browser (so far?)!
